src/Asset/Asset.js

```diff
@@ -30,18 +30,17 @@ export class Asset {
   async filter(filters) {
     const { status, type, sort, pagination } = filters.sanitize()
     const tx_status = txUtils.TRANSACTION_STATUS.confirmed
-
     const [assets, total] = await Promise.all([
       db.query(
         SQL`SELECT model.*, row_to_json(pub.*) as publication
-          FROM ${raw(Publication.tableName)} as pub
-          JOIN ${raw(this.tableName)} as model ON model.id = pub.asset_id
-          WHERE status = ${status}
-            AND tx_status = ${tx_status}
-            AND type = ${type}
-            AND ${PublicationQueries.whereisActive()}
-          ORDER BY pub.${raw(sort.by)} ${raw(sort.order)}
-          LIMIT ${raw(pagination.limit)} OFFSET ${raw(pagination.offset)}`
+      FROM ${raw(Publication.tableName)} as pub
+      JOIN ${raw(this.tableName)} as model ON model.id = pub.asset_id
+      WHERE tx_status = ${tx_status}
+        AND type = ${type}
+        AND ${PublicationQueries.whereisActive()}
+        ${PublicationQueries.andHasStatus(status)}
```

Comment:
This is 100% correct, but I think it helps readability and composition if we keep it as `PublicationQueries.hasStatus(status)` so we can use it later in an OR query if we like and we can continue reading left to right here in the query:

```javascript
WHERE tx_status = ${tx_status}
    AND type = ${type}
    AND ${PublicationQueries.whereisActive()}
    AND ${PublicationQueries.hasStatus(status)}
```

what do you think?

Reply:
👍